index.html

```diff
@@ -38,18 +38,25 @@ <h2 data-i18n="app.title"></h2>
             <div class='col12' id='address-step'>
                 <div class='center pad2'>
                     <span id='instructions' data-i18n="step1.desc"></span>
-                    <p id="couldnt-find" style='display: none;' data-i18n="[html]step1.fail"></p>
                 </div>
                 <div class='col12'>
                     <div class='pad1'>
                         <form id='find'>
                             <input class='col8' id='address' placeholder='' data-i18n="[placeholder]step1.addr"/>
                             <button class='col4' id='findme' data-i18n="step1.button"></button>
+                            <br />
+                            Or <a id="use_my_location" href="#" onclick="return false">use my current location</a>.
```

Comment:
Right now it should fail and display the "Cant find address" error, allowing the user to focus on the address search; though I haven't tested that super well.

Reading through https://developers.google.com/web/updates/2016/04/geolocation-on-secure-contexts-only it might be a bit hard to detect without automatically prompting the user to grant geolocation position permissions on page load, which... seems like it may scare off one or two users.
I can get window.location I suppose; but it might be better to automatically redirect to https via javscript rather than hide